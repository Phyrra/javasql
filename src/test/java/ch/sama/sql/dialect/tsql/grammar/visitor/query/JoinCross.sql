select 1
from a
cross join b on a.id = b.id