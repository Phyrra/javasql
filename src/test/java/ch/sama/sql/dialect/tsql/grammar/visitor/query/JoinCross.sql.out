SELECT 1
FROM [a]
CROSS JOIN [b] ON [a].[id] = [b].[id]